src/controllers/AmazonController.js

```diff
@@ -35,15 +35,10 @@ controller = {
   getAlbumArt: function() {
     var img = document.querySelector("#nowPlayingSection .albumImage");
     if (!img) return "";
-    return img.src
+	return img.src && img.src.replace('45_', '500_');
```

Comment:
I'll probably move that to a BasicController soon, I think the project will
survive ;)

On Tue, May 19, 2015 at 10:24 AM Eric Nemchik <notifications@github.com>
wrote:

> In src/controllers/AmazonController.js
> <https://github.com/msfeldstein/chrome-media-keys/pull/45#discussion_r30623698>
> :
>
> > @@ -35,15 +35,10 @@ controller = {
> >    getAlbumArt: function() {
> >      var img = document.querySelector("#nowPlayingSection .albumImage");
> >      if (!img) return "";
> > -    return img.src
> > +	return img.src && img.src.replace('45_', '500_');
>
> ZOMG I did a tab again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/msfeldstein/chrome-media-keys/pull/45/files#r30623698>
> .
>